qutip/nonmarkov/bofin.py

```diff
@@ -26,6 +28,79 @@
 from qutip.ui.progressbar import BaseProgressBar
 
 
+class BathMode:
+    """
+    Represents a single excitation mode within a bath.
```

Comment:
Thanks! I updated these names and fleshed out the documentation a bit.

I kept the name `modes` for `Bath.modes` because `Bath.exps` looked odd to me. I'm happy to change it to something better -- I'm just not sure what the something better is at the moment.